### PR TITLE
Additions

### PR DESCRIPTION
**Pull request by zkayns**
- Added tags to mod.json
- Added source link to mod.json
- Added descriptions to hacks that didn't have any
- Added README.md (used OpenHack's README as base)
- Added No (C) Mark (from Prism, fixed bug where original level id would show as 1)
- Added Disable Shaders (from Prism)
- Added Transparent Lists (from Crystal)
- Change name of Level Copy Bypass to Copy Bypass
- Moved Practice Music from Global to Cosmetic, the reason being that it doesn't affect the game outside of the level

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    </a>
+    <a href="https://github.com/EclipseMenu/EclipseMenu/issues">
+      <img src="https://img.shields.io/github/issues/EclipseMenu/EclipseMenu" alt="Issues">
+    </a>
+    <a href="">
+      <img src="https://img.shields.io/discord/0?label=discord&logo=discord" alt="Discord">
+    </a>
+    <a href="https://www.buymeacoffee.com">
+      <img src="https://img.shields.io/badge/-buy_me_a%C2%A0coffee-gray?logo=buy-me-a-coffee" alt="Buy Me A Coffee">
+    </a>
```
Comment (FireMario211) [inline README.md:18]: ?

File: mod.json [hunk 2]
```diff
@@ -27,6 +31,11 @@
 			"id": "ninxout.crystal_client",
 			"version": "*",
 			"importance": "superseded"
+		},
+		{
+			"id": "firee.prism",
+			"version": "*",
+			"importance": "superseded"
```
Comment (FireMario211) [inline mod.json:38]: not exactly

**Comment by zkayns** [comment]
thanks for suggestion

**Comment by FireMario211** [comment]
why had no one merged this

**Pull request merged**